packages/veto-app/shared/db/src/gun/index.ts

```diff
@@ -0,0 +1,5 @@
+if (typeof window !== 'undefined') {
+  module.exports = require('./client').default
```

Comment:
Looking at both `client.ts` & `server.ts` it seems like we will bundle server code in the FE.

I'm wondering what need to be shared here? I remember we talked about sharing types, is there anything else we need to share between packages?

Reply:
changed for `__webpack_require__`

Reply:
This change does not address the initial issue. This package still holds back-end scripts that will be present in the front-end build and vice versa.

We should extract what is server specific and leave it in the back-end package. Same for the front-end code, its should be offloaded to the front-end package.